CondTools/L1Trigger/interface/L1ConfigOnlineProdBase.h

```diff
@@ -169,9 +173,9 @@ bool L1ConfigOnlineProdBase<TRcd, TData>::getObjectKey(const TRcd& record, std::
 
   // If L1TriggerKey is invalid, then all configuration objects are
   // already in ORCON.
-  edm::ESHandle<L1TriggerKey> key;
+  const edm::ESHandle<L1TriggerKey> key;
   try {
-    keyRcd.get(key);
+    key = record.getHandle(l1TriggerKeyToken_);
```

Comment:
instead of this changing to record, could you modify the lines https://github.com/cms-sw/cmssw/blob/9aca42dc39d6d2783da2c5d64ef381810494e69c/CondTools/L1Trigger/interface/L1ConfigOnlineProdBase.h#L74 and TRcd& record to use `TRcd& iRecord`? Just to be consistent